Utils/Extensions/NSURLExtensions.swift

```diff
@@ -96,6 +96,24 @@ extension NSURL {
         }
         return nil
     }
+    
+    public func absoluteStringWithoutProtocol() -> String? {
```

Comment:
This makes perfect sense. I thought of only removing www and http/https but we should also remove subdomains and this will be good way to remove it.